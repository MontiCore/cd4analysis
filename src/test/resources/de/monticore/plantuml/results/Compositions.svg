<?xml version="1.0" encoding="us-ascii" standalone="no"?><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" contentStyleType="text/css" height="200px" preserveAspectRatio="none" style="width:208px;height:200px;background:#FFFFFF;" version="1.1" viewBox="0 0 208 200" width="208px" zoomAndPan="magnify"><defs/><g><rect fill="none" height="27.6094" id="_legend" style="stroke:#000000;stroke-width:1.0;" width="120" x="40" y="12"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="110" x="45" y="31.5332">.Compostions CD</text><!--class A--><g id="elem_A"><rect codeLine="19" fill="#FFFFFF" height="27.6094" id="A" style="stroke:#181818;stroke-width:0.5;" width="15" x="74" y="59.6094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="9" x="77" y="79.1426">A</text></g><!--class B--><g id="elem_B"><rect codeLine="21" fill="#FFFFFF" height="27.6094" id="B" style="stroke:#181818;stroke-width:0.5;" width="15" x="124" y="59.6094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="9" x="127" y="79.1426">B</text></g><!--class C--><g id="elem_C"><rect codeLine="23" fill="#FFFFFF" height="27.6094" id="C" style="stroke:#181818;stroke-width:0.5;" width="16" x="73.5" y="147.6094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="10" x="76.5" y="167.1426">C</text></g><!--class D--><g id="elem_D"><rect codeLine="25" fill="#FFFFFF" height="27.6094" id="D" style="stroke:#181818;stroke-width:0.5;" width="16" x="124.5" y="147.6094"/><text fill="#000000" font-family="sans-serif" font-size="14" lengthAdjust="spacing" textLength="10" x="127.5" y="167.1426">D</text></g><text fill="#888888" font-family="sans-serif" font-size="10" lengthAdjust="spacing" textLength="200" x="0" y="190.5996">generated with MontiCore using PlantUML</text><!--SRC=[RKz1QiGm33p7hz0BsZvWAUmcbq8NIalfsUkeZbZR2h92sPRzU-quUoXK1scOqOZHBkibuAFfNB0v3sIzs0W_fcl4Yhg3TMSllAInvy027nCfLk6rSP9GuC5q0DVQ0dfCOx72Ml-xc_RtG6MkPdU_PPfeH70aBc3366UzGCHuGicchESpfTdMn6lQlQhGQL5ySfpUw1lBpIsCyWn2VjBYkkivpfoLE6Ne1rFrjcRQqtkpWSCDz3Smc5eEavR4dynr52UALHpXYtI28oUbyXF2aYbvU0aswVln-HS0]--></g></svg>
