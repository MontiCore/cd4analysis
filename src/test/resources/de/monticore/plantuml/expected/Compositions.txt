@startuml
  skinparam classAttributeIconSize 0
  skinparam legend {
    BorderColor black
    BackGroundColor white
  }
  legend top right
    .Compostions CD
  end legend
  class A

  class B

  class C

  class D

  class E




  center footer generated with MontiCore using PlantUML
@enduml
